include/readline.h

```diff
@@ -4,6 +4,7 @@
 #include <stdio.h>
 
 char *read_line(FILE *file);
+char *peek_line(FILE *file, int offset, long *position);
```

Comment:
Can you rename `offset` to `line_offset` so that it's obvious it's a number of lines? (vs number of bytes with `position`)